outreach/views/order.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from decimal import Decimal
+from flask import request, jsonify, make_response
+from coaster.views import load_models
+from .. import app
+from utils import xhr_only, cors
+from ..models import db, LineItem, SaleItem, ItemCollection, User, Order, OrderSession
+from ..forms import LineItemForm, BuyerForm, OrderSessionForm
+from outreach.mailclient import send_confirmation_mail
+
+
+def jsonify_line_items(line_items):
+    """
+    Serializes and return line items in the format:
+    {item_id: {'quantity': Y, 'final_amount': Z}}
+    """
+    items_json = dict()
+    for line_item in line_items:
+        item = SaleItem.query.get(line_item.item_id)
```

Comment:
To avoid hitting the database multiple times in a loop, do (a) a join query when loading `line_items`, or (b) query a list using the `IN` operator: `SaleItem.query.filter(SaleItem.id.in_([item.item_id for item in line_items])).all()`